cbf/v1/cbf-v1.0.0.yaml

```diff
@@ -0,0 +1,591 @@
+          required:
+            - total
+            - totalPlugs
+        unitsDG:
+          type: object
+          properties:
+            total:
+              type: integer
+              description: Total units DG
+            totalPlugs:
+              type: integer
+              description: Subtotal units DG with plugs (reefer)
+          required:
+            - total
+            - totalPlugs
+      required:
+        - location
+        - equipmentSizeCode
+        - averageGrossWeight
+    EmptyContainer:
+      title: EmptyContainer
+      x-stoplight:
+        id: 2xjaqamt03lnq
+      type: object
+      description: A CBF specification for a particular container type.
+      properties:
+        equipmentSizeCode:
+          type: string
+          enum:
+            - '20'
+            - '40'
+            - '45'
+            - 20HC
+            - 40HC
+            - 45HC
+          example: '20'
+          description: A CBF specification for a particular equipment size and type.
+        averageTotalContainerWeight:
+          type: number
+          format: float
+          description: Cargo Gross Weight plus Container Tare Weight.
+          x-stoplight:
+            id: 9x4mxy8qv6kvi
+        units:
+          type: integer
+          description: Total units non DG
+        discharge:
+          type: array
+          description: Lists all the `EMPTY` containers being discharged.
+          items:
+            $ref: '#/components/schemas/DischargeEmptyContainer'
+      required:
+        - equipmentSizeCode
+        - averageTotalContainerWeight
+        - units
+        - discharge
+    DischargeEmptyContainer:
+      title: Discharge Empty Container
+      type: object
+      description: A CBF specification for a particular container type.
+      x-stoplight:
+        id: 3870fdc4937be
+      properties:
+        portVisitReference:
+          type: string
+          maxLength: 50
+          pattern: ^\S+(\s+\S+)*$
+          example: NLRTM1234589
+          description: The unique reference that can be used to link different transportCallReferences to the same port visit. The reference is provided by the port to uniquely identify a port call.
+        transportCallReference:
+          type: string
+          description: "A carrier definied reference to a TransportCall.\r\n\r\nIn the case the Means of Transport is a Vessel and the facility is a Port/Terminal - this reference should be considered a Terminal Call Reference\r\n"
+          example: N987G4567
+          pattern: ^\S+(\s+\S+)*$
+          maxLength: 100
+        equipmentSizeCode:
+          type: string
+          enum:
+            - '20'
+            - '40'
+            - '45'
+            - 20HC
+            - 40HC
+            - 45HC
+          example: '20'
+          description: A CBF specification for a particular equipment size and type.
+        units:
+          type: integer
+          description: Total units non DG
```

Comment:
Is it necessary to specify that this is non DG as it is in the empty container object ;-)

Reply:
For consistency purposes.